src/frontend/src/FindBox.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import IconButton from '@material-ui/core/IconButton';
+import SearchIcon from '@material-ui/icons/Search';
+import AddBoxIcon from '@material-ui/icons/AddBox';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Map from './Map';
+import Find from './FindBox';
+
+const axios = require('axios');
+
+class FindBox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      usernum : this.props.userNum,
+      users : [],
+      areas : []
+    };
+  };
+
+  handleChange = (e) => {
+    this.props.changeLoc({
+      name: e.target.name,
+      value: e.target.value,
+      index: e.target.id
+    });
+  }
+
+  makeInput() {
+    let output = [];
+
+    for (let i =0; i < this.props.userNum; i++) {
+      output.push(
+        <div>
+          <Paper>
+            <TableCell>
+              <Typography paragraph='true' >
+            <p> {'user' + (i+1)} </p>
+            <div>
+              <div> latitude : <input type="text" name="lat" id={i} value={this.props.users[i].location.lat} onChange={this.handleChange} /></div>
+              <div> longitude : <input type="text" name="lng" id={i} value={this.props.users[i].location.lng} onChange={this.handleChange} /></div>
+              <select onChange={this.handleChange} id ={i} name="transportation">
+                <option value="public">public</option>
+                <option value="driving">driving</option>
+              </select>
+              <button onClick={() => this.props.selectMarker(i)}>Select location</button>
+            </div>
+              </Typography>
+            </TableCell>
+          </Paper>
+          <br></br>
+        </div>
+      )
+    }
+
+    return output;
+  };
+
+  showResult() {
```

Comment:
showAreaResult or showLocationResult

need to name it clearer that makes we know what type of data this function returns.